### PR TITLE
Issue #283

### LINKED ISSUE #283: Slashify Interviewer
**Describe the problem and the impact.**
_Is there an issue you want to solve? Who does this feature impact? How many people will this feature impact? Will this feature be used often? What will happen if we *don't* implement this feature? Please clearly and concisely describe the problem you are facing._

There are lots of interviewer commands that aren't slashified

**Describe your proposed solution(s).**
_Please provide us with as much detail as possible and feel free to suggest more than one solution if there are alternative solutions. We may follow back up to clarify details we aren't sure of._
Slashify them by replicating what was done for `coin`

**Deadlines**
_Do you want it done by a certain date? Describe how urgently you need the feature complete. Please be specific and provide additional context if needed. This will help us with prioritization._
Probably by next week

**Additional Information**
_Feel free to add screenshots or any other contextual information we should know for building this feature._
I'm down to work on this

### PR DESCRIPTION
**Pull request by KuroganeToyama**
## Summary of Changes
- Slashified `interviewer` command

## Related Issues
- Resolve #283

### CODE REVIEW

File: src/commandDetails/interviewer/clear.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  await clearProfile(id);
+  return 'Your interviewer profile has been cleared!';
+};
+
+export const interviewerClearCommandDetails: CodeyCommandDetails = {
+  name: 'clear',
+  aliases: ['clr'],
+  description: 'Clear interviewer data',
+  detailedDescription: `**Examples:**
+\`${container.botPrefix}interviewer clear\``,
```
Comment (Fan-Yang-284) [inline src/commandDetails/interviewer/clear.ts:32]:
```suggestion
\`${container.botPrefix}interviewers clear\``,
```

File: src/commandDetails/interviewer/profile.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    { name: '**Domains**', value: _.isEmpty(domains) ? 'None' : getDomainsString(domains) },
+  ]);
+
+  return { embeds: [profileEmbed] };
+};
+
+export const interviewerProfileCommandDetails: CodeyCommandDetails = {
+  name: 'profile',
+  aliases: ['pf'],
+  description: 'Modify profile data',
```
Comment (Fan-Yang-284) [inline src/commandDetails/interviewer/profile.ts:44]:
```suggestion
  description: 'Display profile data',
```

File: src/commandDetails/interviewer/list.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  const interviewersToShow = shuffledInterviewers.slice(0, RESULTS_PER_PAGE);
+  // Get information from each interviewer
+  const interviewersInfo = await Promise.all(
+    interviewersToShow.map((interviewer) => getInterviewerDisplayInfo(interviewer)),
+  );
+
+  // If there's no data, then interviewersInfo is an array of undefined, which messed things up
+  // Thus need to display something else instead of interviewersInfo.join()
+  const embedDescription =
+    interviewersInfo[0] === undefined ? 'No data to display' : interviewersInfo.join();
```
Comment (Fan-Yang-284) [inline src/commandDetails/interviewer/list.ts:56]:
```suggestion
    interviewersInfo[0] === undefined ? 'No data to display' : interviewersInfo.join('');
```

**Review by Fan-Yang-284 — CHANGES_REQUESTED** [review]
Awesome work! Thanks for working on this - I think just change these two things

- currently the `list` command produces slightly odd-looking output, so just remove the comma created from `.join`
image
- maybe modify the description for profile since it only displays information
BTW you can ignore the above suggestion of adding an 's' to the example usage, that was me being dumb and not realising they were aliases.

**Review by Fan-Yang-284 — APPROVED** [review]

**Review by probro27 — APPROVED** [review]
@KuroganeToyama LGTM! Thank you for figuring this out! The PR is good to go

**Pull request merged**